WebViewControl.Avalonia/WebView.Avalonia.cs

```diff
@@ -92,7 +92,9 @@ partial class WebView : BaseControl {
         /// <paramref name="isSystemEvent">True if is a system focus event, or false if is a navigation</paramref>
         /// </summary>
         protected virtual bool OnSetFocus(bool isSystemEvent) {
-            var focusedElement = TopLevel.GetTopLevel(this).FocusManager.GetFocusedElement();
+            // Null check is done for TopLevel as VisualRoot can be null in case when
+            // user detaches tab and proxy is enabled and there is a dummy webview created to set proxy information

```

Comment:
please rephrase, this is a public repo... none of this makes sense in a public context

Reply:
Thanks for pointing this out. I have updated it.